qemu/tests/cfg/x86_cpu_flags.cfg

```diff
@@ -48,6 +48,21 @@
                             expect_items = 48
                             cpu_model_flags = ",-la57"
                             no_flags = "la57"
+                - intel_pt:
```

Comment:
Does it need `only HostCpuVendor.intel`?

Reply:
Oh you have it above, ignore me.

Reply:
> Does it need `only HostCpuVendor.intel`?

it is added in Line 11.